great_expectations_provider/operators/great_expectations.py

```diff
@@ -96,7 +96,8 @@ def __init__(
         ] = data_context_root_dir
         self.data_context_config: DataContextConfig = data_context_config
         self.checkpoint_name: Optional[str] = checkpoint_name
-        self.checkpoint_config: Optional[CheckpointConfig] = checkpoint_config or {}
+        self.checkpoint_config: Optional[CheckpointConfig] = checkpoint_config or {
```

Comment:
just curious, was this change generated by a code formatter? otherwise i would just leave it in the single line?

Reply:
Oh, probably. I don't remember changing that by hand. I can reformat.